src/pal/src/configure.cmake

```diff
@@ -933,6 +933,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(PAL_PT_READ_D PT_READ_D)
   set(PAL_PT_WRITE_D PT_WRITE_D)
   set(HAS_FTRUNCATE_LENGTH_ISSUE 1)
+  set(HAVE_SCHED_OTHER_ASSIGNABLE 1)
```

Comment:
Well this is debatable whether this cmake configuration (or in general should it) conform with POSIX standard's UB clauses or just logical distribution of compile/linker switches. But that was just me suggesting to have fewer lines of code change. :)